apps/meteor/client/lib/chats/flows/uploadFiles.ts

```diff
@@ -61,7 +61,9 @@ export const uploadFiles = async (chat: ChatAPI, files: readonly File[], resetFi
 						return;
 					}
 
-					const shouldConvertSentMessages = e2eRoom.shouldConvertSentMessages({ msg });
+					room.encrypted ? e2eRoom.resume() : e2eRoom.pause();
```

Comment:
did we starting executing code like this? I've always suggested to do the following, it is much cleaner IMO.. ternary should be used only attributions..
```suggestion
					if (room.encrypted) {
						e2eRoom.resume();
					} else {
						e2eRoom.pause();
					}
```

Reply:
+1 from me. Goes for both ternaries and &&s, ||s (room.encrypted && e2eRoom.resume()). 

Also reduces more noise down the road in diffs.

Reply:
Agreed. Done!

In my defense, I copied this piece of code from someplace else, hence the negligence. 😬 

Will keep this in mind moving forward, thanks!